lib/slack/web/config.rb

```diff
@@ -1,3 +1,8 @@
+begin
+  require 'openssl'
+rescue LoadError # rubocop:disable Lint/HandleExceptions
+end
```

Comment:
I don't think this is a great place to put this, `lib/slack-ruby-client.rb` is better, it has all the `require`.

Reply:
@dblock Agree. Will move it there.